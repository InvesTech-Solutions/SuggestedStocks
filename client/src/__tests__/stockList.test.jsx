import React from 'react';
import StockList from '../components/stockList.jsx';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';



test('renders', () => {
    const wrapper = shallow(<StockList />);

    expect(toJson(wrapper)).toMatchSnapshot();
})