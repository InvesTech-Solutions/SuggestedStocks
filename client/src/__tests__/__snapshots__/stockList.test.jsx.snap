// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  id="cssCarousel"
>
  <div
    className="left"
  />
  <div
    id="currentScene"
  >
    <div
      className="slide"
      style={
        Object {
          "MozTransition": "all 0.3s ease-in-out",
          "OTransition": "all 0.3s ease-in-out",
          "WebkitTransition": "all 0.3s ease-in-out",
          "marginLeft": "-0px",
          "transition": "all 0.3s ease-in-out",
        }
      }
    >
      <Stock />
      <Stock />
      <Stock />
      <Stock />
      <Stock />
      <Stock />
      <Stock />
      <Stock />
      <Stock />
      <Stock />
      <Stock />
      <Stock />
    </div>
  </div>
  <div
    className="right"
  >
    <a
      href="#"
      onClick={[Function]}
    >
       
      <svg
        height="17"
        viewBox="0 0 11 17"
        width="11"
      >
        <g
          fillRule="evenodd"
          transform="rotate(45 1.243 3)"
        >
          <rect
            height="3"
            width="12"
          />
          <rect
            height="12"
            width="3"
            x="9"
          />
        </g>
      </svg>
    </a>
  </div>
</div>
`;
